modules/ssl/ssl_engine_config.c

```diff
@@ -697,7 +702,11 @@ const char *ssl_cmd_SSLCryptoDevice(cmd_parms *cmd,
 #endif
     else {
         err = "SSLCryptoDevice: Invalid argument; must be one of: "
+#if MODSSL_USE_OPENSSL_STORE
+              "'builtin' (none), 'provider' (use OpenSSL >= 3.0 provider STORE)";
+#else
               "'builtin' (none)";
+#endif
```

Comment:
Maybe:
```
        err = "SSLCryptoDevice: Invalid argument; must be one of: "
              "'builtin' (none)"
#if MODSSL_USE_OPENSSL_STORE
              ", 'provider' (use OpenSSL >= 3.0 provider STORE)"
#endif
              ;
```
for better "evolutivity"?

Reply:
sure, will change.
I didn't like the lonely semicolon at the end :-)